adobe/iomanip.hpp

```diff
@@ -16,6 +16,8 @@
 
 #include <adobe/config.hpp>
 
+#include <adobe/iomanip_fwd.hpp>
```

Comment:
If header order matters, you'd better leave a big-@ss comment telling people not to move the header.  Should really refer to a bug report and say that it's a workaround.